CHANGELOG.md

```diff
@@ -79,6 +80,84 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.13.0
+
+### Change
+
+- refactor(ai-proxy): move read_response into ai_driver.request function [#12101](https://github.com/apache/apisix/pull/12101)
+- refactor: mcp server framework implementation #[12168](https://github.com/apache/apisix/pull/12168)
```

Comment:
This is a refactoring of the `mcp-bridge` plugin https://github.com/apache/apisix/pull/12151. It is not a breaking change for users. In the previous version, the plugin does not even exist.